### PR TITLE
Added tests for api-response and user-response reducers

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
-Added tests for api-response and user-response reducers ( Didn't add tests for all other reducers to avoid big PR)
-added mocks
- configured user model
@hemantjadon  @vibhcool @djmgit Please review :)

### CODE REVIEW

File: src/app/reducers/api-response.spec.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import * as fromapiresponse from './api-response';
+import * as apiAction from '../actions/api';
+import * as paginationAction from '../actions/pagination';
+import { ApiResponse, ApiResponseResult, ApiResponseMetadata, ApiResponseUser, ApiResponseAggregations } from '../models/api-response';
```
Comment (hemantjadon) [inline src/app/reducers/api-response.spec.ts]: separate out in different lines

File: src/app/reducers/api-response.spec.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import { MockApiResponse } from '../shared/mocks/feedItem.mock';
+
+describe('ApiResponseReducer', () => {
+	describe('undefined action', () => {
+		it('should return the default state', () => {
+			const action = {} as any;
+
+			const result = fromapiresponse.reducer(undefined, action);
```
Comment (hemantjadon) [inline src/app/reducers/api-response.spec.ts]: use camel case fromApiResponse -> do this all the places

File: src/app/reducers/api-response.spec.ts (hunk 1 above)
Comment (Achint08) [inline src/app/reducers/api-response.spec.ts]: Done! (Y)

File: src/app/reducers/api-response.spec.ts (hunk 2 above)
Comment (Achint08) [inline src/app/reducers/api-response.spec.ts]: Done. :)

**Pull request merged**